Translate high-availability into Bahasa Indonesia

evanap opened the pull request:
Hi!

I have translated the page `/docs/setup/production-environment/tools/kubeadm/high-availability/` into Bahasa Indonesia.

This PR addresses #22296.

sftim commented on content/id/docs/tasks/tools/kubeadm/high-availability.md:
```diff
@@ -0,0 +1,364 @@
+menggabungkan Node _control plane_ satu persatu.
+- Jangan lupakan bahwa kunci dekripsi dari `--certificate-key` akan kadaluarsa setelah dua jam, pada pengaturan semula.
+
+## Tugas-tugas umum setelah _bootstrapping_ _control plane_
```
```suggestion
## Tugas-tugas umum setelah _bootstrap_ _control plane_
```
?

evanap replied:
Thanks, I decided to adopt it. @sftim

sftim commented on content/id/docs/tasks/tools/kubeadm/high-availability.md:
````diff
@@ -0,0 +1,364 @@
+sudo kubeadm join 192.168.0.200:6443 --token 9vr73a.a8uxyaju799qwdjv --discovery-token-ca-cert-hash sha256:7c2e69131a36ae2a042a339b33381c6d0d43887e2de83720eff5359e26aec866
+```
+
+## Distribusi sertifikat manual {#manual-certs}
````
Also fine:
```suggestion
## Distribusi sertifikat manual
```

evanap replied:
Decided to remove it as well.

sftim commented on content/id/docs/tasks/tools/kubeadm/high-availability.md:
```diff
@@ -0,0 +1,364 @@
+
+    {{< caution >}}
+    Salinlah hanya sertifikat yang berada pada daftar di atas saja. Perkakas kubeadm akan mengambil alih pembuatan sertifikat lainnya
+    dengan SANs yang dibutuhkan untuk Node _control-plane_ yang akan bergabung. Jika kamu menyalin seluruh sertifikat tanpa sengaja,
```
nit:
```suggestion
    dengan SANs yang dibutuhkan untuk Node _control plane_ yang akan bergabung. Jika kamu menyalin seluruh sertifikat tanpa sengaja,
```

evanap replied:
Thanks, removed all hyphens for control plane.

girikuncoro commented on content/id/docs/tasks/tools/kubeadm/high-availability.md:
```diff
@@ -0,0 +1,364 @@
+---
+title: Membuat Klaster dengan Ketersediaan Tinggi menggunakan kubeadm
```
nit: need consistency here: membuat or membangun (from the _index.md)

evanap replied:
Ah, I updated the index but forgot about the title. Fixed in bc5c6c5.

girikuncoro commented on content/id/docs/tasks/tools/kubeadm/high-availability.md:
```diff
@@ -0,0 +1,364 @@
+Lihat juga [dokumentasi pembaruan](/id/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-15).
+
+{{< caution >}}
+Laman ini tidak menunjukkan cara untuk menjalankan klastermu pada penyedia layanan _cloud_. Pada _environment cloud_, kedua pendekatan yang didokumentasikan di sini tidak akan bekerja untuk objek Service dengan tipe LoadBalancer maupun PersistentVolume dinamis.
```
nit: cloud is not italicized in RSNI
```suggestion
Laman ini tidak menunjukkan cara untuk menjalankan klastermu pada penyedia layanan cloud. Pada _environment_ cloud, kedua pendekatan yang didokumentasikan di sini tidak akan bekerja untuk objek Service dengan tipe LoadBalancer maupun PersistentVolume dinamis.
```

evanap replied:
Alright, unitalicized all occurrences of "cloud" in bc5c6c5.

girikuncoro commented on content/id/docs/tasks/tools/kubeadm/high-availability.md:
```diff
@@ -0,0 +1,364 @@
+Untuk kedua metode kamu membutuhkan infrastruktur seperti berikut:
+
+- Tiga mesin yang memenuhi [kebutuhan minimum kubeadm](/id/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#before-you-begin) untuk
+  Node _master_
```
we have replaced "master" with _control plane_, because it's kind of banned by tech community. I saw the rest of page using _control plane_ already
```suggestion
  Node _control plane_
```

evanap replied:
Ah, it's one of the non-inclusive words... Fixed in bc5c6c5.

girikuncoro commented on content/id/docs/tasks/tools/kubeadm/high-availability.md:
```diff
@@ -0,0 +1,364 @@
+
+- Tiga mesin yang memenuhi [kebutuhan minimum kubeadm](/id/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#before-you-begin) untuk
+  Node _master_
+- Tiga mesin yang memenuhi [kebutuhan minimum kubeadm](/id/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#before-you-begin) untuk Node _worker_
```
this is the correct anchor link https://kubernetes.io/id/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#sebelum-mulai

evanap replied:
Fixed in bc5c6c5.

girikuncoro commented on content/id/docs/tasks/tools/kubeadm/high-availability.md:
```diff
@@ -0,0 +1,364 @@
+
+Untuk kedua metode kamu membutuhkan infrastruktur seperti berikut:
+
+- Tiga mesin yang memenuhi [kebutuhan minimum kubeadm](/id/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#before-you-begin) untuk
```
this is the correct anchor link https://kubernetes.io/id/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#sebelum-mulai

evanap replied:
Fixed in bc5c6c5.

girikuncoro commented on content/id/docs/tasks/tools/kubeadm/high-availability.md:
```diff
@@ -0,0 +1,364 @@
+
+1.  Buat sebuah _load balancer_ kube-apiserver dengan sebuah nama yang yang akan mengubah ke dalam bentuk DNS.
+
+    - Pada _environment cloud_ kamu harus meletakkan Node _control plane_ di belakang _load balancer_ yang meneruskan TCP. _Load balancer_ ini mendistribusikan trafik ke seluruh Node _control plane_ pada daftar tujuan. _Health check_ untuk
```
```suggestion
    - Pada _environment_ cloud kamu harus meletakkan Node _control plane_ di belakang _load balancer_ yang meneruskan TCP. _Load balancer_ ini mendistribusikan trafik ke seluruh Node _control plane_ pada daftar tujuan. _Health check_ untuk
```

evanap replied:
Fixed in bc5c6c5.

girikuncoro commented on content/id/docs/tasks/tools/kubeadm/high-availability.md:
```diff
@@ -0,0 +1,364 @@
+      apiserver adalah pengujian TCP pada porta yang didengarkan oleh kube-apiserver
+      (nilai semula `:6443`).
+
+    - Tidak direkomendasikan untuk menggunakan alamat IP secara langsung pada _environment cloud_.
```
```suggestion
    - Tidak direkomendasikan untuk menggunakan alamat IP secara langsung pada _environment_ cloud.
```

evanap replied:
Fixed in bc5c6c5.

girikuncoro commented on content/id/docs/tasks/tools/kubeadm/high-availability.md:
````diff
@@ -0,0 +1,364 @@
+    nc -v LOAD_BALANCER_IP PORT
+    ```
+
+    - Kegalatan koneksi yang ditolak memang diantisipasi karena apiserver belum
````
oh nice one https://kbbi.kemdikbud.go.id/entri/galat

evanap replied:
Yep! It's usually used to translate "error".

girikuncoro commented on content/id/docs/tasks/tools/kubeadm/high-availability.md:
```diff
@@ -0,0 +1,364 @@
+
+3.  Tambahkan Node _control plane_ lainnya pada grup tujuan _load balancer_.
+
+## Node _control plane_ dand etcd bertumpuk (_stacked_)
```
typo?
```suggestion
## Node _control plane_ dan etcd bertumpuk (_stacked_)
```

evanap replied:
My bad, fixed in bc5c6c5.

girikuncoro commented on content/id/docs/tasks/tools/kubeadm/high-availability.md:
```diff
@@ -0,0 +1,364 @@
+
+    - Opsi `--upload-certs` digunakan untuk mengunggah sertifikat-sertifikat yang harus dibagikan ke seluruh
+      Node _control-plane_ pada klaster. Jika sebaliknya, kamu memilih untuk menyalin sertifikat ke
+      seluruh Node _control-plane_ sendiri atau menggunakan perkakas automasi, silakan hapus opsi ini dan merujuk ke bagian [Distribusi sertifikat manual](#manual-certs) di bawah.
```
if you decide to remove the alias anchor here (https://github.com/kubernetes/website/pull/22669/files#diff-6f6c6f2a2b1a160e3bf2cac7899fe8c6R282), this needs change

evanap replied:
Removed the alias and updated the anchor in bc5c6c5.

girikuncoro commented on content/id/docs/tasks/tools/kubeadm/high-availability.md:
```diff
@@ -0,0 +1,364 @@
+
+    {{< note >}}
+    Beberapa _plugin_ jaringan CNI membutuhkan pengaturan tambahan, seperti menentukan CIDR IP untuk Pod, meski beberapa lainnya tidak.
+    Lihat [dokumentasi jaringan CNI](/id/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#pod-network).
```
this is the correct anchor link https://kubernetes.io/id/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#jaringan-pod

evanap replied:
Fixed in bc5c6c5.

girikuncoro commented on content/id/docs/tasks/tools/kubeadm/high-availability.md:
```diff
@@ -0,0 +1,364 @@
+    {{< /caution >}}
+
+2.  Pasang _plugin_ CNI pilihanmu:
+    [Ikuti petunjuk berikut](/id/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#pod-network)
```
incorrect anchor link

evanap replied:
Fixed in bc5c6c5.

girikuncoro commented on content/id/docs/tasks/tools/kubeadm/high-availability.md:
```diff
@@ -0,0 +1,364 @@
+
+2.  Pasang _plugin_ CNI pilihanmu:
+    [Ikuti petunjuk berikut](/id/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#pod-network)
+    untuk menginstal penyedia CNI. Pastikan konfigurasinya corresponds dengan CIDR Pod yang ditentukan pada berkas konfigurasi kubeadm jika diterapkan.
```
translate "corresponds" as well

evanap replied:
Sorry I missed that. Fixed in bc5c6c5.

girikuncoro commented on content/id/docs/tasks/tools/kubeadm/high-availability.md:
```diff
@@ -0,0 +1,364 @@
+
+### Memasang klaster etcd
+
+1.  Ikuti [petunjuk berikut](/id/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm/) untuk membangun klaster etcd.
```
we don't have this yet, I think we can fallback to EN page https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm/

evanap replied:
Reverted to original EN page in bc5c6c5.

girikuncoro commented on content/id/docs/tasks/tools/kubeadm/high-availability.md:
```diff
@@ -0,0 +1,364 @@
+
+1.  Ikuti [petunjuk berikut](/id/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm/) untuk membangun klaster etcd.
+
+2.  Lakukan pengaturan SSH seperti yang dijelaskan [di sini](#manual-certs).
```
if you decide to remove the alias anchor here (https://github.com/kubernetes/website/pull/22669/files#diff-6f6c6f2a2b1a160e3bf2cac7899fe8c6R282), this needs change

evanap replied:
Yep, fixed in bc5c6c5.

evanap commented:
Thanks for the reviews! @sftim @girikuncoro

I've addressed it all. Feel free to check it again.

Also, I'm sorry that I made a mistake placing the page into the wrong path. I've fixed it, it should be in the correct path now.

girikuncoro commented on content/id/docs/setup/production-environment/tools/kubeadm/high-availability.md:
```diff
@@ -0,0 +1,364 @@
+
+### Memasang klaster etcd
+
+1.  Ikuti [petunjuk berikut](/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm/) untuk membangun klaster etcd.
```
@evanap sorry my mistake, there's actually a fallback mechanism when referring to doc pages that don't have localization yet, so this is fine if we add `/id` prefix
```suggestion
1.  Ikuti [petunjuk berikut](/id/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm/) untuk membangun klaster etcd.
```

girikuncoro replied:
you can verify this in the netlify preview page @evanap

evanap replied:
I made a follow-up PR for this, it's here #22709 @girikuncoro
It doesn't show any sign of fallback mechanism though. I checked with the netlify preview page of the PR but it returns 404 instead of the EN page 🤔

girikuncoro commented:
@evanap all looking good, thanks for this, feel free to take this as follow up PR https://github.com/kubernetes/website/pull/22669#discussion_r459881651

/lgtm
/approve

girikuncoro commented:
Sorry about the confusion, it's indeed there's no fallback mechanism yet.
We ended up writing scripts for replacing links with existing ID pages only
here

https://github.com/jk8s/sig-docs-id/blob/master/scripts/localize_url.py

On Sat, Jul 25, 2020, 8:38 AM Evan <notifications@github.com> wrote:

> *@evanap* commented on this pull request.
> ------------------------------
>
> In
> content/id/docs/setup/production-environment/tools/kubeadm/high-availability.md
> <https://github.com/kubernetes/website/pull/22669#discussion_r460348453>:
>
> > +    sudo kubeadm join 192.168.0.200:6443 --token 9vr73a.a8uxyaju799qwdjv --discovery-token-ca-cert-hash sha256:7c2e69131a36ae2a042a339b33381c6d0d43887e2de83720eff5359e26aec866 --control-plane --certificate-key f8902e114ef118304e561c3ecd4d0b543adc226b7a07f675f56564185ffe0c07
>
> +    ```
>
> +
>
> +    - Opsi `--control-plane` menunjukkan `kubeadm join` untuk membuat _control plane_ baru.
>
> +    - Opsi `--certificate-key ...` akan membuat sertifikat _control plane_ diunduh
>
> +      dari Secret `kubeadm-certs` pada klaster dan didekripsi menggunakan kunci yang diberikan.
>
> +
>
> +## Node etcd eksternal
>
> +
>
> +Membangun sebuah klaster dengan Node etcd eksternal memiliki prosedur yang mirip dengan etcd bertumpuk
>
> +dengan pengecualian yaitu kamu harus setup etcd terlebih dulu, dan kamu harus memberikan informasi etcd
>
> +pada berkas konfigurasi kubeadm.
>
> +
>
> +### Memasang klaster etcd
>
> +
>
> +1.  Ikuti [petunjuk berikut](/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm/) untuk membangun klaster etcd.
>
>
> I made a follow-up PR for this, it's here #22709
> <https://github.com/kubernetes/website/pull/22709> @girikuncoro
> <https://github.com/girikuncoro>
> It doesn't show any sign of fallback mechanism though. I checked with the
> netlify preview page of the PR but it returns 404 instead of the EN page
> 🤔
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/website/pull/22669#discussion_r460348453>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAYLMG7MKUGRLPF4NMJVMDTR5IZSRANCNFSM4PFXWOMQ>
> .
>

evanap commented:
Alright, I closed the follow-up PR
